parm/use_cases/met_tool_wrapper/EnsembleStat/EnsembleStat.conf

```
@@ -222,21 +188,13 @@ ENSEMBLE_STAT_OUTPUT_FLAG_RELP = BOTH
 #ENSEMBLE_STAT_OUTPUT_FLAG_PRC = BOTH
 #ENSEMBLE_STAT_OUTPUT_FLAG_ECLV = BOTH
 
-
-ENSEMBLE_STAT_ENSEMBLE_FLAG_LATLON = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_MEAN = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_STDEV = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_MINUS = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_PLUS = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_MIN = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_MAX = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_RANGE = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_VLD_COUNT = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_FREQUENCY = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_NEP = FALSE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_NMEP = FALSE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_RANK = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_WEIGHT = FALSE
+ENSEMBLE_STAT_NC_ORANK_FLAG_LATLON = TRUE
+ENSEMBLE_STAT_NC_ORANK_FLAG_MEAN = TRUE
+ENSEMBLE_STAT_NC_ORANK_FLAG_RAW = TRUE
+ENSEMBLE_STAT_NC_ORANK_FLAG_RANK = TRUE
+ENSEMBLE_STAT_NC_ORANK_FLAG_PIT = TRUE
+ENSEMBLE_STAT_NC_ORANK_FLAG_VLD_COUNT = TRUE
+ENSEMBLE_STAT_NC_ORANK_FLAG_WEIGHT = TRUE
```

```suggestion
ENSEMBLE_STAT_NC_ORANK_FLAG_WEIGHT = FALSE
```
Previously, the weight was set to FALSE but has been changed to TRUE here. This certainly isn't a big deal but would be more consistent if left as FALSE.